tests/utils/test_resource_manager.py

```diff
@@ -0,0 +1,25 @@
+"""
+Copyright (c) 2022 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+import pytest
+from unittest import mock
```

Comment:
flexmock?

Reply:
is it issue? I'm not so familiar with it - I'm trying to change it to fm but it's not so intuitive I would say